README.md

```diff
@@ -1680,6 +1680,7 @@ _Related: [Static site generators](#static-site-generators)_
 - [Accent](https://www.accent.reviews/) - Open-source, self-hosted, developer-oriented translation tool. ([Source Code](https://github.com/mirego/accent)) `BSD-3-Clause` `Elixir`
 - [Localizer](https://localizer.dev) - Free self-hosted open-source crowd-translating service for your product. ([Demo](https://localize.todorant.com), [Source Code](https://github.com/backmeupplz/localizer-backend)) `MIT` `Nodejs/Docker`
 - [Pootle](https://pootle.translatehouse.org) - Online translation and localization tool. ([Source Code](https://github.com/translate/pootle)) `GPL-3.0` `Python`
+- [Tolgee](https://tolgee.io) - Developer & translator firendly web-based localization platform enabling users to translate directly in the app they develop. ([Source Code](https://github.com/tolgee/server)) `Apache-2.0` `Kotlin/TypeScript`
```

Comment:
- Typo `firendly`

>  The Language tag is the main server-side requirement for the software. Don't include frameworks or specific dialects.


- The only supported installation method is Docker -> please change  `Kotlin/TypeScript` to `Docker`